### PR TITLE
WXS-2565: Fix dependency updates

### PR DESCRIPTION
**Pull request by saschazar21**
Updated the following dependencies:
- **Babel** (incl. presets) to **v7**
- **Webpack** to **v4**

Fixed eslint errors (e.g. object destructuring)
Fixed webpack production config (missing css/scss loader ruleset)
Updated documentation to fix typos, or enhance description wherever necessary

### CODE REVIEW

**Comment by saschazar21** [comment]
Jira Issue Link: [https://jira.xxxlgroup.com/browse/WXS-2565](https://jira.xxxlgroup.com/browse/WXS-2565)

File: 01-node-yarn-package-json/Readme.md [hunk 1]
```diff
@@ -66,18 +69,18 @@ Open the `package.json` file and **remove** the line `"main": "index.js",`, the
 
 **Create** an `index.js` file containing `console.log('Hello world');`
```
Comment (rohrlaf) [inline 01-node-yarn-package-json/Readme.md]:
```suggestion
- **Create** an `index.js` file containing `console.log('Hello world');`
```

File: 01-node-yarn-package-json/Readme.md [hunk 2]
```diff
@@ -66,18 +69,18 @@ Open the `package.json` file and **remove** the line `"main": "index.js",`, the
...
 
-* **Run:** `node .` in this folder (`index.js` is the default file Node looks for in a folder). It should print:
-
-    Hello world
+- **Run:** `node .` in this folder (`index.js` is the default file Node looks for in a folder). It should print:
```
Comment (rohrlaf) [inline 01-node-yarn-package-json/Readme.md]:
```suggestion
- **Run** `node .` in this folder (`index.js` is the default file Node looks for in a folder). It should print:
```

File: 01-node-yarn-package-json/Readme.md [hunk 3]
```diff
@@ -89,19 +92,20 @@ In `package.json`, **add** a `scripts` section:
 }
 ```
 
-`start` is the name we give to the *task* that will run our program. We are going to create a lot of different tasks in this `scripts` object throughout this tutorial. `start` is typically the name given to the default task of an application. Some other standard task names are `stop` and `test`.
+`start` is the name we give to the _task_ that will run our program. We are going to create a lot of different tasks in this `scripts` object throughout this tutorial. `start` is typically the name given to the default task of an application. Some other standard task names are `stop` and `test`.
 
 `package.json` must be a valid JSON file, which means you cannot have trailing commas. So be careful when manually editing your `package.json` file.
 
-* **Run:** `yarn start`
+- **Run:** `yarn start`
```
Comment (rohrlaf) [inline 01-node-yarn-package-json/Readme.md]:
```suggestion
- **Run** `yarn start`
```

File: 01-node-yarn-package-json/Readme.md [hunk 4]
```diff
@@ -89,19 +92,20 @@ In `package.json`, **add** a `scripts` section:
...
 
-    Hello world
+  Hello world
 
 ## Git and `.gitignore`
 
 **Initialize** a Git repository with `git init`. At the end of every page we'll remind you to commit your changes. If you want to know more about git read [this article](http://rogerdudler.github.io/git-guide/).
```
Comment (rohrlaf) [inline 01-node-yarn-package-json/Readme.md]:
```suggestion
- **Initialize** a Git repository with `git init`.

At the end of every page we'll remind you to commit your changes. If you want to know more about git read [this article](http://rogerdudler.github.io/git-guide/).
```

File: 01-node-yarn-package-json/Readme.md [hunk 5]
```diff
@@ -89,19 +92,20 @@ In `package.json`, **add** a `scripts` section:
...
 
 **Create** a `.gitignore` file and add the following to it:
```
Comment (rohrlaf) [inline 01-node-yarn-package-json/Readme.md]:
```suggestion
- **Create** a `.gitignore` file and add the following to it:
```

File: 01-node-yarn-package-json/Readme.md [hunk 6]
```diff
@@ -117,15 +121,16 @@ In this section we will install and use a package. A "package" is simply a piece
 
 - **Install** the community-made package called `color` by running `yarn add color`.
 
-Open `package.json` to see how Yarn automatically added `color` in  `dependencies`.
+Open `package.json` to see how Yarn automatically added `color` in `dependencies`.
 
 A `node_modules` folder has been created to store the package.
 
 **Add** `node_modules/` to your `.gitignore`
```
Comment (rohrlaf) [inline 01-node-yarn-package-json/Readme.md]:
```suggestion
- **Add** `node_modules/` to your `.gitignore`
```

File: 01-node-yarn-package-json/Readme.md [hunk 7]
```diff
@@ -117,15 +121,16 @@ In this section we will install and use a package. A "package" is simply a piece
...
 
-You will also notice that a `yarn.lock` file was generated by Yarn. You should commit this file to your repository, as it will ensure that everyone on your team uses the same version of your packages. If you're sticking to NPM instead of Yarn, the equivalent of this file is the *shrinkwrap*.
+You will also notice that a `yarn.lock` file was generated by Yarn. You should commit this file to your repository, as it will ensure that everyone on your team uses the same version of your packages. If you're sticking to NPM instead of Yarn, the equivalent of this file is the _shrinkwrap_.
 
 **Replace** the content of your `index.js` file:
```
Comment (rohrlaf) [inline 01-node-yarn-package-json/Readme.md]:
```suggestion
- **Replace** the content of your `index.js` file:
```

File: 01-node-yarn-package-json/Readme.md [hunk 8]
```diff
@@ -134,15 +139,15 @@ const redHexa = color({ r: 255, g: 0, b: 0 }).hex();
 console.log(redHexa);
 ```
 
-* **Run:** `yarn start`
+- **Run:** `yarn start`
```
Comment (rohrlaf) [inline 01-node-yarn-package-json/Readme.md]:
```suggestion
- **Run** `yarn start`
```

File: 01-node-yarn-package-json/Readme.md [hunk 9]
```diff
@@ -134,15 +139,15 @@ const redHexa = color({ r: 255, g: 0, b: 0 }).hex();
...
 
-    #FF0000
+  #FF0000
 
 Congratulations, you installed and used a package!
 
 `color` is just used in this section to teach you how to use a simple package. We won't need it anymore, so you can uninstall it:
 
-* **Run:** `yarn remove color`
+- **Run:** `yarn remove color`
```
Comment (rohrlaf) [inline 01-node-yarn-package-json/Readme.md]:
```suggestion
- **Run** `yarn remove color`
```

File: 02-babel-es6-eslint-husky/Readme.md [hunk 10]
```diff
@@ -7,32 +7,39 @@ We're now going to use some ES6 syntax, which is a great improvement over the "o
...
+
 ```js
 const str = 'ES6';
 console.log(`Hello ${str}`);
 ```
 
-We're using a *template string* here, which is an ES6 feature that lets us inject variables directly inside the string without concatenation using `${}`. Note that template strings are created using **backquotes**.
+We're using a _template string_ here, which is an ES6 feature that lets us inject variables directly inside the string without concatenation using `${}`. Note that template strings are created using **backquotes**.
+
+- **Run:** `yarn add --dev @babel/node` to install the `babel-node` CLI executable for Babel.
```
Comment (rohrlaf) [inline 02-babel-es6-eslint-husky/Readme.md]:
```suggestion
- **Run** `yarn add --dev @babel/node` to install the `babel-node` CLI executable for Babel.
```

File: 02-babel-es6-eslint-husky/Readme.md [hunk 11]
```diff
@@ -7,32 +7,39 @@ We're now going to use some ES6 syntax, which is a great improvement over the "o
...
+- **Run:** `yarn add --dev @babel/preset-env` to install a Babel preset package called `env`, which contains configurations for the most recent ECMAScript features supported by Babel.
 
 **Create** a `.babelrc` file at the root of your project, which is a JSON file for your Babel configuration. Insert the following snippet to make Babel use the `env` preset:
 
 ```json
 {
-  "presets": [
-    "env"
-  ]
+  "presets": ["env"]
```
Comment (rohrlaf) [inline 02-babel-es6-eslint-husky/Readme.md]: See `02-babel-es6-eslint-husky/.babelrc`
```suggestion
  "presets": ["@babel/preset-env"]
```

**Pull request merged**